tox.ini

```diff
@@ -375,6 +379,17 @@ setenv =
 setenv =
     {[django_autopatch]setenv}
 
+
+[django_restframework]
+setenv =
+    DATADOG_ENV = test
+    DJANGO_SETTINGS_MODULE = restframework.settings
+
```

Comment:
I use a different app than for other django unit tests because using _rest_framework_ would break most of our django unit tests. 

This way we can only test in a specific app that we properly trace exceptions raised during _rest_framework_ view processing.